src/utils/common.js

```diff
@@ -17,3 +17,24 @@ export const formatTime = (date) => {
   const minutes = castTimeFormat(date.getMinutes());
   return `${hours}:${minutes}`;
 };
+
+export const sortArray = (property, order = `-1`) => {
```

Comment:
Ну такое себе решение с `-1`
Уж лучше бы
```suggestion
export const sortArray = (property, asc = false) => {
```